helper/testhelpers/docker/testhelpers.go

```diff
@@ -509,13 +509,13 @@ func BuildContextFromTarball(reader io.Reader) (BuildContext, error) {
 				break
 			}
 
-			return nil, fmt.Errorf("failed to parse provided tarball: %v")
+			return nil, fmt.Errorf("failed to parse provided tarball: %v", err)
 		}
 
 		data := make([]byte, int(header.Size))
 		read, err := archive.Read(data)
 		if err != nil {
-			return nil, fmt.Errorf("failed to parse read from provided tarball: %v")
+			return nil, fmt.Errorf("failed to parse read from provided tarball: %v", err)
```

Comment:
Oh, I wonder if we should wrap this error instead with `%w`?

Reply:
Oops, sorry, I had a response queued up and didn't hit "Comment". `%v` works for me! If we need to fix this, we can get all of the errors in one fell swoop.

Reply:
No worries! Sorry for jumping the gun a little on merging, I just didn't want this PR to go on for too long - and I figured we'd collectively be happy either way, so long as it was fixed.